scripts/usb_reset.py

```diff
@@ -212,17 +212,17 @@ def setup_cgroup(domid, pid):
 
     try:
         # unbuffered write to ensure each one is flushed immediately
-        with open(cg_dir + "/tasks", "w", 0) as tasks, \
-                open(cg_dir + "/devices.deny", "w", 0) as deny, \
-                open(cg_dir + "/devices.allow", "w", 0) as allow:
+        with open(cg_dir + "/tasks", "wb") as tasks, \
+                open(cg_dir + "/devices.deny", "wb") as deny, \
+                open(cg_dir + "/devices.allow", "wb") as allow:
 
             # deny all
-            deny.write("a")
+            deny.write(b"a")
 
             # grant rw access to /dev/null by default
-            allow.write(get_ctl("/dev/null", "rw"))
+            allow.write(get_ctl("/dev/null",b"rw"))
```

Comment:
And keep the space for readability / standard formatting:
```suggestion
            allow.write(get_ctl("/dev/null", b"rw"))
```